src/api/controller/front/dev.js

```diff
@@ -1,30 +1,46 @@
 import Koa from 'koa';
 import koaWebpack from 'koa-webpack';
+import copyCustomization from './copyCustomization';
 
-import webpackConfig from '../../../app/webpack.config.babel';
+let customizationApplied = false;
+let koaWebpackMiddleware = null;
 
 export default () => {
     const app = new Koa();
 
-    app.use(koaWebpack({
-        config: webpackConfig,
-        dev: {
-            publicPath: webpackConfig.output.publicPath,
-            headers: {
-                'Content-Type': 'text/html; charset=utf-8',
-            },
-            stats: {
-                colors: true,
-            },
-            quiet: false,
-            noInfo: true,
-        },
-        hot: {
-            log: console.log,
-            path: '/__webpack_hmr',
-            heartbeat: 10 * 1000,
-        },
-    }));
+    app.use(async (ctx, next) => {
+        if (!customizationApplied) {
+            customizationApplied = true;
+
+            await copyCustomization();
+        }
+
+        const webpackConfig = require('../../../app/webpack.config.babel').default; // eslint-disable-line
```

Comment:
Had to dynamically require the file, otherwise the calls to `path.resolve` inside it wouldn't succeed as the index.html is not yet copied